tools/chat_helpers/channel_msg_8ball.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+function shared_channel_msg_8ball() {
+	static $answers = array(
+		'Signs point to yes.',
+		'Yes.',
+		'Reply hazy, try again.',
+		'Without a doubt.',
+		'My sources say no.',
+		'As I see it, yes.',
+		'You may rely on it.',
+		'Concentrate and ask again.',
+		'Outlook not so good.',
+		'It is decidedly so.',
+		'Better not tell you now.',
+		'Very doubtful.',
+		'Yes - definitely.',
+		'It is certain.',
+		'Cannot predict now.',
+		'Most likely.',
+		'Ask again later.',
+		'My reply is no.',
+		'Outlook good.',
+		'Don\'t count on it.'
+	);
+
+	return $answers[rand(0, count($answers) - 1)];
```

Comment:
I was going to suggest `array_rand` for this, but apparently it has dodgy randomness - wtf php